### DISCUSSION TITLE
Feature: show popover preview for links to repositories

### DISCUSSION THREAD

**Discussion by unbyte** [community/community#7188] [Other Feature Feedback, Questions, & Ideas]
![image](https://user-images.githubusercontent.com/5772358/140030408-be157bd5-915f-4f9f-a87b-5757766676c4.png)

We have long been used to moving the mouse over the issue or PR link to view its information, but there has been no such thing for the links of repositories.

I suddenly have this idea because I found https://github.com/actions/create-release from Google, but it was deprecated and listed some alternatives, so I have to open all links of alternatives to get which one is the most popular.

**Comment by kidonng** [comment]
Hovercard for repositories is already enabled on some pages, like user profiles. It is just not enabled _everywhere_.

However it is easy to tell GitHub to show previews for them via [a user script](https://github.com/kidonng/cherry/tree/master/scripts#github-hovercards) I just wrote:
<img width="502" alt="2021-11-06 23 27 33" src="https://user-images.githubusercontent.com/44045911/140614969-02a50161-5761-4764-a45d-e9dfb1518ec1.png">

(I also submitted [a feature request to Refined GitHub](https://github.com/refined-github/refined-github/issues/5052), if you are interested.)
